tools/cldr-code/src/main/java/org/unicode/cldr/tool/GenerateDateTimeTestData.java

```diff
@@ -513,11 +513,11 @@ private static ImmutableMap<Object, Object> getTestCaseForZonedDateTime(
                         dateTimeGluePatternFormatType,
                         icuServiceBuilder);
 
-        // "input_string" = the ISO 18601 UTC time zone formatted string of the zoned date time
-        optionsBuilder.put("inputString", zdt.toString());
+        // "input" = the ISO 18601 UTC time zone formatted string of the zoned date time
```

Comment:
ISO - International Organization for Standardization

ISO 18601:2013 - Packaging and the environment — General requirements for the use of ...

Reply:
```suggestion
        // "input" = the ISO 8601 UTC time zone formatted string of the zoned date time
```